docker-compose.yml

```diff
@@ -89,6 +89,8 @@ services:
     restart: unless-stopped
     ports:
       - 8088:8088
+      # When in cypress-mode ->
+      - 8081:8081
```

Comment:
NOTE: debated whether to try and open this port conditionally, but seemed impossible and researched whether there's downside in opening a port-to-nowhere in other scenarios and I seemed there's no downside. Could also just run Cypress against 8088 too if we prefer

Reply:
Is it fairly simple to run Cypress against 8088? I know that we can change the port when running locally.

Reply:
yup it's super easy, I'm just unclear if we want segmentation on PORT similar to the segmentation we have at the database level. In theory would could have both a dev and cypress environment running in parallel and not conflicting on ports ... Though I don't think you can fire two docker-compose at the same time (?)

Reply:
One thing that's kind of good with using a different port, is that firing up Cypress against `docker compose up` will fail as it's looking for Superset on 8081, forcing the user to stop docker-compose and to fire it up properly `CYPRES_CONFIG=true docker-compose up`, meaning it enforces having the right backend, with the right test sets and examples loaded up, preventing some confusion around using the wrong backend.